src/assets/text/how-to-play.ts

```diff
@@ -0,0 +1,86 @@
+// TODO: add missing links
+export const howToPlay = {
+  headings: {
```

Comment:
What about having this in lower case and adding a style to show them in all caps?